newIDE/app/src/SceneEditor/ExtractAsExternalLayoutDialog.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import * as React from 'react';
+import FlatButton from '../UI/FlatButton';
+import Text from '../UI/Text';
+import TextField from '../UI/TextField';
+import { ColumnStackLayout } from '../UI/Layout';
+import Dialog, { DialogPrimaryButton } from '../UI/Dialog';
+import HelpButton from '../UI/HelpButton';
+
+type Props = {|
+  suggestedName: string,
+  onApply: (chosenName: string) => void,
+  onCancel: () => void,
+|};
+
+export default function ExtractAsExternalLayoutDialog({
+  suggestedName,
+  onApply,
+  onCancel,
+}: Props) {
+  const [chosenName, setChosenName] = React.useState<string>(suggestedName);
+
+  const apply = React.useCallback(() => onApply(suggestedName), [
+    onApply,
+    suggestedName,
+  ]);
+
+  return (
+    <Dialog
+      title={<Trans>Extract as an external layout</Trans>}
+      actions={[
+        <FlatButton
+          key="cancel"
+          label={<Trans>Cancel</Trans>}
+          primary={false}
+          onClick={onCancel}
+        />,
+        <DialogPrimaryButton
+          key="apply"
+          label={<Trans>Apply</Trans>}
+          primary={true}
+          onClick={apply}
+        />,
+      ]}
+      secondaryActions={[
+        <HelpButton
+          helpPagePath="/interface/scene-editor/external-layouts/"
+          key="help"
+        />,
+      ]}
+      onRequestClose={onCancel}
+      onApply={apply}
+      open
+      maxWidth="sm"
+      exceptionallyStillAllowRenderingInstancesEditors
+    >
+      <ColumnStackLayout noMargin>
+        <Text>
+          <Trans>
+            Selected objects will be moved to a new external layout.
```

Comment:
```suggestion
            Selected instances will be moved to a new external layout. External layouts can be created during the game using events.
```

Reply:
I think it's better not to add the 2nd sentence because people won't read it. Adding too much text is the same as removing it all.
Also, I think that in addition of being too much, it's also not enough. If people end up here just to look around, I prefer that they click on the help button and read the full documentation page rather than think they understood but didn't. For instance, the page explains which action to use to create the instances.

Reply:
Ok!